src/components/pages/Bookings/PreFilters/PreFilters.jsx

```diff
@@ -9,19 +9,13 @@ import FilterByBookingPeriod from './FilterByBookingPeriod'
 import FilterByEventDate from './FilterByEventDate.jsx'
 import FilterByVenue from './FilterByVenue'
 
-const PreFilters = ({ offerVenue }) => {
+const PreFilters = ({ offerVenueId, applyPreFilters }) => {
   // eslint-disable-next-line no-unused-vars
-  const [appliedFilters, setAppliedFilters] = useState({
-    bookingBeginningDate: EMPTY_FILTER_VALUE,
-    bookingEndingDate: EMPTY_FILTER_VALUE,
-    offerDate: EMPTY_FILTER_VALUE,
-    offerVenue: offerVenue || ALL_VENUES,
-  })
   const [selectedFilters, setSelectedFilters] = useState({
     bookingBeginningDate: EMPTY_FILTER_VALUE,
     bookingEndingDate: getToday(),
     offerDate: EMPTY_FILTER_VALUE,
-    offerVenue: offerVenue || ALL_VENUES,
+    offerVenueId: offerVenueId || ALL_VENUES,
```

Comment:
```suggestion
    offerVenueId: offerVenueId,
```
Si on le met en valeur par défaut dans les propTypes, on devrait pouvoir l'enlever ici